test/cmd/authentication.sh

```diff
@@ -91,10 +92,11 @@ users:
 - name: valid_token_user
   user:
     exec:
-      apiVersion: client.authentication.k8s.io/v1beta1
+      apiVersion: client.authentication.k8s.io/v1
       command: echo
       args:
-        - '{"apiVersion":"client.authentication.k8s.io/v1beta1","status":{"token":"admin-token"}}'
+        - '{"apiVersion":"client.authentication.k8s.io/v1","status":{"token":"admin-token"}}'
+      interactiveMode: IfAvailable
```

Comment:
I would like to see a failing v1 test here for when the mode is left unspecified.

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/c1e9e36256f871cd811706552f919b8b4f9c1c09 adds a negative v1 test